frontend/src/components/commons/ProgressPane.jsx

```diff
@@ -19,42 +21,50 @@ class ProgressPane extends Component {
   changeNode(diff) {
     const index = this.state.currentNode + diff;
     this.setState({ currentNode: index, currentBody: this.props.progressSpecs[index].body });
+    scrollToTop();
   }
 
   render() {
     return (
-      <div className="progress-pane emib-text-zone">
-        <div aria-label="progress" className="step-indicator">
-          <ul className="steps">
-            {this.props.progressSpecs.map(tab => (
-              <ProgressNode
-                key={tab.id}
-                id={tab.id}
-                text={tab.text}
-                current={this.state.currentNode}
-              />
-            ))}
-          </ul>
-        </div>
-        {this.props.paneTitle ? <h1 className="progress-pane">{this.props.paneTitle}</h1> : null}
-        {this.state.currentBody}
-        {this.state.currentNode < this.props.progressSpecs.length - 1 && (
-          <div className="centeredButtons">
-            <button type="button" className="btn btn-primary" onClick={() => this.changeNode(1)}>
-              {LOCALIZE.commons.nextButton}
-            </button>
-          </div>
-        )}
-        {this.state.currentNode === this.props.progressSpecs.length - 1 && (
-          <div className="centeredButtons">{this.props.exitButton}</div>
-        )}
-        {this.state.currentNode > 0 && (
-          <div className="centeredButtons">
-            <button type="button" className="btn btn-secondary" onClick={() => this.changeNode(-1)}>
-              {LOCALIZE.commons.backButton}
-            </button>
+      <div>
+        <div className="progress-pane emib-text-zone">
+          <div aria-label="progress" className="step-indicator">
+            <ul className="steps">
+              {this.props.progressSpecs.map(tab => (
+                <ProgressNode
+                  key={tab.id}
+                  id={tab.id}
+                  text={tab.text}
+                  current={this.state.currentNode}
+                />
+              ))}
+            </ul>
           </div>
-        )}
+          {this.props.paneTitle ? <h1 className="progress-pane">{this.props.paneTitle}</h1> : null}
```

Comment:
Here's a cleaner way to write this:
```
{this.props.paneTitle && <h1 className="progress-pane">{this.props.paneTitle}</h1>}
```

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/87/commits/45e4c95a8be9ede2051a9fb26abd7401ed2e3a48